src/sass/components/_about.scss

```diff
@@ -0,0 +1,28 @@
+.container 
```

Comment:
klasa container już istniej w _global.scss

Reply:
Na poczatku w html bedziesz mial ze mna duzo pracy mysle ze juz same css nie bedziesz mial ze mna problemow.
dlatego zrobilem takie bledy np w serokosciach bo one byly na probe, bo nie chcialem popelnic bledu jak w hw 8, odrazu chce aby elementy w mojej sekcji same dobrze sie ukladaly w innych szerokosciach, same z siebie, aby ta moja krowa sama wskoczyla na swoje miejsce na lewo od tekstu w tablecie i desktopie, no i teraz wskakuje. 
Pytanie teraz na wyrost,  Jak mam myslisz aby modyfikowac te klase  container zaleznosci od moich  potrzeb, (nie twierdze ze bedzie taka potrzeba) ale gdyby moja sekcja miala np inne boczne paddingi przy jakiejs szerokosci ekranu, np tabletowej niz sekcje innych osob? To w moim pliku about.css tworze .container--about? czy w pliku global.css w global bylo by schludnie ale rosnie ryzyko konfliktow wersji. takie proste zadaie a tyle pytan sie pojawia, jendak jak nie zaczniesz pisac kodu to nic sie nie nauczysz.

Reply:
container będzie jednakowy dla wszystkich na mobilce i jednakowy dla wszystkich na teblecie i również jednakowy na desktopie. W mobilce container na paddingi boczne po 20px, a w tablecie ustaliłem na 34px (nie wszystkie sekcje tak mają, ale musimy się dostosować do jednej szerokości), więc nie musisz się przejmować odległościami w containerze. Zerknij sobie w _global, klasa container ma padding: 0 20px;, a potem są @media na tablet i zmienia na padding: 0 34px;